default.conf

```diff
@@ -19,9 +19,11 @@ server {
 
     location / {
         index  index.html index.htm;
+        try_files $uri $uri/ /$1/ $uri/index.html /index.html;
     }
 
-    location ~ ^/(\w+/?)+ {
-      try_files $uri $uri/ /$1/ $uri/index.html /index.html;
-    } 
+    location /account/ {
+        autoindex on;
+        alias /usr/share/nginx/html/;
```

Comment:
why isn't the existing location directive working? `/account/` is covered in the Nginx config

Reply:
removed /account block as root path didn't have `try_files` directive